build/packages.targets

```diff
@@ -16,87 +14,88 @@
-        <PackageReference Update="Microsoft.VisualStudio.Telemetry" Version="16.3.59" />
-        <PackageReference Update="Microsoft.VisualStudio.Text.Data" Version="$(VSComponentsVersion)" />
-        <PackageReference Update="Microsoft.VisualStudio.Text.Logic" Version="$(VSComponentsVersion)" />
-        <PackageReference Update="Microsoft.VisualStudio.Text.UI" Version="$(VSComponentsVersion)" />
-        <PackageReference Update="Microsoft.VisualStudio.Text.UI.Wpf" Version="$(VSComponentsVersion)" />
-        <PackageReference Update="Microsoft.VisualStudio.TextManager.Interop" Version="7.10.6071" />
         <PackageReference Update="Microsoft.VisualStudio.TextManager.Interop.10.0" Version="10.0.30319" />
-        <PackageReference Update="Microsoft.VisualStudio.Threading" Version="$(VSThreadingVersion)" />
-        <PackageReference Update="Microsoft.VisualStudio.Threading.Analyzers" Version="$(VSThreadingVersion)" />
-        <PackageReference Update="Microsoft.VisualStudio.Workspace.VSIntegration" Version="16.6.55-preview-0003" />
-        <PackageReference Update="Microsoft.VisualStudio.Utilities" Version="$(VSFrameworkVersion)" />
+        <PackageReference Update="Microsoft.VSSDK.BuildTools" Version="16.8.3038" />
         <PackageReference Update="Microsoft.Web.Xdt" Version="2.1.2" />
         <PackageReference Update="Newtonsoft.Json" Version="$(NewtonsoftJsonPackageVersion)" />
-        <PackageReference Update="System.Collections.Immutable" Version="1.7.1" />
         <PackageReference Update="System.Diagnostics.Debug" Version="$(SystemPackagesVersion)" />
-        <PackageReference Update="System.Security.Cryptography.Pkcs" Version="$(CryptographyPackagesVersion)" />
         <PackageReference Update="System.Security.Cryptography.Cng" Version="$(CryptographyPackagesVersion)" />
+        <PackageReference Update="System.Security.Cryptography.Pkcs" Version="$(CryptographyPackagesVersion)" />
         <PackageReference Update="System.Security.Cryptography.ProtectedData" Version="4.4.0" />
-        <PackageReference Update="System.Threading.Tasks.Dataflow" Version="4.9.0" />
-        <PackageReference Update="VSLangProj" Version="7.0.3300" />
-        <PackageReference Update="VSLangProj110" Version="11.0.61030" />
-        <PackageReference Update="VSLangProj157" Version="15.7.0" />
-        <PackageReference Update="VSLangProj2" Version="7.0.5000" />
         <PackageReference Update="VSSDK.TemplateWizardInterface" Version="12.0.4" />
-        <PackageReference Update="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.8" />
+
+        <!--
+          These packages are installed transitively by the Microsoft.VisualStudio.SDK metapackage,
+          and as a general rule, the versions listed below match what the Microsoft.VisualStudio.SDK metapackage brings in.
+          The packages are listed here explicitly to help resolve NU1604 errors.
```

Comment:
I'm not sure I understand the comment. If they are transitive there should be no need to specify them right?
This is only necessary if they are specified.